src/screens/Details.js

```diff
@@ -0,0 +1,13 @@
+import { render } from "../shared/dom.js";
+const template = `
+<div id ="details-screen" class="details">
+    <template class="details__logo"></template>
+    <template class="details__trailer"></template>
+    <template class="details__gallery"></template>
+    <template class="details__footer"></template>
```

Comment:
Footer? 
We have footer and header in project already, you have to be up to date with main branch